### DISCUSSION TITLE
Quick Postfix main.cf?

### DISCUSSION THREAD

**Discussion by whisperdancer** [simple-login/app#1319] [Self-hosting question]
Your docs suggest replacing all instances of mydomain.com with my actual domain:

Replace `/etc/postfix/main.cf` with the following content. Make sure to replace `mydomain.com` by your domain.

Does the below line need to be changed to append_dot_my real domain or left as is? Assuming my domain is dm.com, would I change the below line to `append_dot_dm` or leave it as is? 

`append_dot_mydomain = no`

**Comment by nguyenkims** [comment]
You should leave it as indicated in the doc, which is `append_dot_mydomain = no`. It's actually the default value in Postfix.
